### PR TITLE
finalizing the events table to take up the correct side of the screen and match the figjam

### PR DESCRIPTION
**Pull request by MGS31**
Ok here is the final pass on the events widget.

Should match the figjam way more, only take up the side of the screen and only requires a few installs to make it work.

Don't forget to npm i! 

Screenshot 2022-08-09 at 3 13 09 PM
m

### CODE REVIEW

File: leafsmart-front-end/src/components/Events.js
Comment (b1u3too) [inline leafsmart-front-end/src/components/Events.js:54]: The only thing I'd suggest tweaking are a couple classes here to get things centred! Looks like the event-widget class name's been phased out by tailwind in full now, so we should be safe to remove it. I tinkered and these classes seem to centre it nicely!: 
image

**Review by b1u3too — APPROVED** [review]
This is GORGEOUS work! I leave it up to our hivemind whether centering right now is worth adding one more push for. :)

File: leafsmart-front-end/package.json
Comment (rachelpr) [inline leafsmart-front-end/package.json:11]: what does moment do?

File: leafsmart-front-end/src/components/EventCard.js
Comment (rachelpr) [inline leafsmart-front-end/src/components/EventCard.js:19]: typo here for "className" 👀

**Comment by rachelpr** [comment]
Screen Shot 2022-08-09 at 5 12 10 PM
On Windsor images are loading weird, and on London they're not loading at all 🤔 

With Andreas suggestion they load, but are a bit small. Do we want to discuss including them at all if they are on the small side?
Screen Shot 2022-08-09 at 5 15 34 PM

File: leafsmart-front-end/src/components/EventCard.js
Comment (MGS31) [reply leafsmart-front-end/src/components/EventCard.js:19]: Cheers this has been fixed and re-submitted

**Comment by MGS31** [comment]
@rachelpr confirming changes were made to sort out the styling issues you had with Windsor per @b1u3too recommendations. All should be good for merging

**Comment by b1u3too** [comment]
> @rachelpr confirming changes were made to sort out the styling issues you had with Windsor per @b1u3too recommendations. All should be good for merging

Excellent job on these updates Matt! I plonked through a good handful of cities and am not seeing any "images for ants" bugs like Rachel experienced. It may be wise to catch this branch up with main before merging if there are any conflicts!

**Review by rachelpr — APPROVED** [review]
another fine job! excited for no ant pictures :D

**Pull request merged**
